packages/amplication-client/src/Resource/ServicesTile.tsx

```diff
@@ -13,28 +13,30 @@ import OverviewSecondaryTile from "./OverviewSecondaryTile";
 import { AppContext } from "../context/appContext";
 import { Resource } from "../models";
 import { GET_MESSAGE_BROKER_CONNECTED_SERVICES } from "../Workspaces/queries/resourcesQueries";
+import { AnalyticsEventNames } from "../util/analytics-events.types";
 
 type Props = {
   resourceId: string;
 };
 
-const EVENT_DATA: TrackEvent = {
-  eventName: "rolesTileClick",
-};
-
 function ServicesTile({ resourceId }: Props) {
   const history = useHistory();
   const { currentWorkspace, currentProject } = useContext(AppContext);
   const getResourceVars = { variables: { where: { id: resourceId } } };
-  const { data, loading, refetch } = useQuery<{ messageBrokerConnectedServices: Resource[] }>(
-    GET_MESSAGE_BROKER_CONNECTED_SERVICES, getResourceVars);
+  const { data, loading, refetch } = useQuery<{
+    messageBrokerConnectedServices: Resource[];
+  }>(GET_MESSAGE_BROKER_CONNECTED_SERVICES, getResourceVars);
   const { trackEvent } = useTracking();
 
   // eslint-disable-next-line
-  useEffect(() => { refetch(getResourceVars) }, []);
+  useEffect(() => {
+    refetch(getResourceVars);
```

Comment:
why refetch on component construct ?

Reply:
this is not related to my pr, this is just the formatting fix